system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs

```diff
@@ -473,6 +508,180 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Ethereum Bridge
+parameter_types! {
+	// The gateway address is set by governance.
+	pub storage EthereumGatewayAddress: H160 = H160::zero();
+}
+
+parameter_types! {
+	pub const CreateAssetCall: [u8;2] = [53, 0];
+	pub const CreateAssetDeposit: u128 = (UNITS / 10) + EXISTENTIAL_DEPOSIT;
```

Comment:
```suggestion
	pub const CreateAssetDeposit: u128 = 10 * UNITS;
```

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/130/commits/0c6e184afa829c55e242623fb923defff61dcf2c.